frontend/src/components/common/iconButton/IconButton.style.ts

```diff
@@ -0,0 +1,39 @@
+import Icon from "../icon/Icon";
+import styled from "styled-components";
+
+export const IconButtonContainer = styled.div`
```

Comment:
컴포넌트 이름도 IconButton인 만큼 div 태그보다는 button 태그로 나타내는 건 어떨까요?
```suggestion
export const IconButtonContainer = styled.button`
```

Reply:
넵 button이라는걸 잘 보여줄 수 있을 것 같네요! 수정하였습니다ㅎㅎ